var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -18,7 +18,10 @@ class Caliper(CMakePackage):
     homepage = "https://github.com/LLNL/Caliper"
     git      = "https://github.com/LLNL/Caliper.git"
 
+    maintainers = ["daboehme"]
+
     version('master')
+    version('2.5.0', tag='v2.5.0')
```

Comment:
can't this be built from tarballs?

Reply:
It can - changed it to use the tarball. Not retroactively applying this to the old versions though.

Adiak needs submodules, so the github-generated tarballs won't work there.

Reply:
Great. It would be nice if the LLNL packages could move towards supporting external dependencies instead of relying on git submodules, cause that way we can verify the checksums & have spack handle the case where the same package occurs multiple times as a dependency of different packages.